test/eyeshade/ads.integration.test.js

```diff
@@ -34,13 +34,11 @@ test('ads payout report cron job takes a snapshot of balances', async t => {
   const paymentId = uuidV4()
   const providerId = uuidV4()
   await wallets.insert({ paymentId, providerId })
-  const insertedWallet = await wallets.findOne({ paymentId: paymentId })
-  console.log('inserted wallet is')
-  console.log(insertedWallet)
 
   // Create an ad transaction so there is a payment_id with a balance
   const txId = uuidV4()
   const createdAt = new Date()
+  createdAt.setMonth(createdAt.getMonth() - 2)
```

Comment:
this does not set the date to `2019-02-01T:00:00:00.000Z` it only modifies the month, so we would have a gap at between each payout depending on if one failed or if one was late or otherwise